src/browser.js

```diff
@@ -65,20 +69,34 @@ class BrowserModal extends Modal {
         // Call super (calls props.onToggle)
         super.onToggle(state, props);
 
-        // Add body class
+        // Add body class and padding to scrollbar.
         if (typeof document !== 'undefined') {
             const container = document.body;
 
-            // Toggle open class
-            toggleClass(container, 'modal-open', state);
-
-            // Update body padding (for scrollbar)
+            // Add model when opening.
```

Comment:
That change would tie the variable name into the comment which means that any future refactor will likely introduce a mystery comment bug that confuses the programmer more than informing them. The only purpose of this comment was to highlight that the adding and removing are performed in separated places within the same function.